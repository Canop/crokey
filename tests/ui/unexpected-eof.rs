fn main() {
    crokey::key!();
    crokey::key!(ctrl);
    crokey::key!(ctrl-);
}
