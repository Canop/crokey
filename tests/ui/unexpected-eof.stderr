error: unexpected end of input, expected one of: character literal, integer literal, identifier
 --> tests/ui/unexpected-eof.rs:2:5
  |
2 |     crokey::key!();
  |     ^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `$crate::__private::key` which comes from the expansion of the macro `crokey::key` (in Nightly builds, run with -Z macro-backtrace for more info)

error: expected `-`
 --> tests/ui/unexpected-eof.rs:3:5
  |
3 |     crokey::key!(ctrl);
  |     ^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `$crate::__private::key` which comes from the expansion of the macro `crokey::key` (in Nightly builds, run with -Z macro-backtrace for more info)

error: unexpected end of input, expected one of: character literal, integer literal, identifier
 --> tests/ui/unexpected-eof.rs:4:5
  |
4 |     crokey::key!(ctrl-);
  |     ^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `$crate::__private::key` which comes from the expansion of the macro `crokey::key` (in Nightly builds, run with -Z macro-backtrace for more info)
